.circleci/config.yml

```diff
@@ -341,40 +332,48 @@ workflows:
       - scan-docker-image:
           requires:
             - rspec-tests
+      - cleanup_merged:
+          name: cleanup_merged_live
+          context:
+            - laa-claim-non-standard-magistrates-fee-dev-crm4
+            - laa-claim-non-standard-magistrates-fee-dev-crm4
 
   build-push-branch:
     jobs:
-      - branch-build-approval:
-          type: approval
-          filters:
-            branches:
-              ignore:
-                - main
       - build-to-ecr:
           context: laa-claim-non-standard-magistrates-fee-dev
-          requires:
-            - branch-build-approval
       - hold-dev:
           type: approval
           requires:
             - build-to-ecr
-      - deploy-dev:
+      - deploy-to-environment:
+          name: deploy_dev
+          environment: dev
+          dynamic_hostname: false
           context: laa-claim-non-standard-magistrates-fee-dev
           requires:
             - hold-dev
-      - hold-dev-crm4:
-          type: approval
-          requires:
-            - build-to-ecr
-      - deploy-dev-crm4:
+#      - hold-dev-crm4:
+#          type: approval
+#          requires:
+#            - build-to-ecr
+      - deploy-to-environment:
```

Comment:
I think part of the benefit of dynamic releases is we don't need a "dev-crm4" environment at all - it was a sticking plaster put in place to deal with not having enough dev environments, so I'm hoping we can delete every reference to it in this PR.

Reply:
I also agree with this - I was using it as a testing ground instead of messing up everyone's dev. Eventually it would go.